src/modules/Packages.rb

```diff
@@ -847,13 +847,7 @@ def InitializeAddOnProducts
           "Found list of add-on products to preselect: %1",
           @add_on_products_list
         )
-        if Mode.auto
-          Builtins.y2warning("This is an AutoYaST installation. "\
-            "Only Add-on products will be added which have been defined "\
-            "in the add-on section of the AutoYaST configuration file.")
-        else
-          AddOnProduct.AddPreselectedAddOnProducts(@add_on_products_list)
-        end
+        AddOnProduct.AddPreselectedAddOnProducts(@add_on_products_list)
```

Comment:
This will also add the products from `/add_on_products.xml` or (plain text `/add_on_products`) file.

This should be documented and possibly mentioned in the release notes (an incompatible change).